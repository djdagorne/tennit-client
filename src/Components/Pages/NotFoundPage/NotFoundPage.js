import React, {Component} from 'react';

/* 
Upon a page breaking error or a invalid URL request I reroute users to this small 404 page.
*/

class NotFoundPage extends Component {
    render() {
		return (
			<div className="content-container">
				<div>
					<h2>Page not found</h2>
					<p>Try going back to your previous page.</p>
				</div>
			</div>
		)
    }
}

export default NotFoundPage;