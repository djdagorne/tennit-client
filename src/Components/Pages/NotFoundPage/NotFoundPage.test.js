import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import NotFoundPage from './NotFoundPage'
import '../../../setupTest'



it('renders without crashing', () => {
        const wrapper = shallow(<NotFoundPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
})
