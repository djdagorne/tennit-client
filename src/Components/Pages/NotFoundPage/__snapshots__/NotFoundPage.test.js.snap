// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="content-container"
>
  <div>
    <h2>
      Page not found
    </h2>
    <p>
      Try going back to your previous page.
    </p>
  </div>
</div>
`;
